distro/run/core/src/test/java/org/camunda/bpm/run/test/config/deploy/DeployChangedOnlyUnsetTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.run.test.config.deploy;
+
+import org.camunda.bpm.run.CamundaBpmRunProcessEngineConfiguration;
+import org.camunda.bpm.run.test.AbstractRestTest;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DeployChangedOnlyUnsetTest extends AbstractRestTest {
+
+  @Autowired
+  private CamundaBpmRunProcessEngineConfiguration engineConfig;
+
+  @Test
+  public void shouldEnableDeployChangedOnlyOnCamundaRunProperty() {
+    assertThat(engineConfig.isDeployChangedOnly()).isEqualTo(true);
```

Comment:
**Covers**: The defaulting of the `deploy-changed-only` property ([ref](https://github.com/camunda/camunda-bpm-platform/blob/83db22ab702c6e67cf03a4dff1516e99dc389860/distro/run/core/src/main/java/org/camunda/bpm/run/CamundaBpmRunConfiguration.java#L58)); if the `:true` suffix is changed, the test will fail